services/github/src/handlers.js

```diff
@@ -504,17 +490,47 @@ async function statusHandler(message) {
     `Attempting to update status of the checkrun for ${organization}/${repository}@${sha} (${taskState.conclusion})`,
   );
   try {
+    const taskDefinition = await this.queueClient.task(taskId);
+    const {customCheckRun} = (taskDefinition.extra && taskDefinition.extra.github) || {};
+    const {textArtifactName} = customCheckRun || CUSTOM_CHECKRUN_TEXT_ARTIFACT_NAME;
+
+    let customCheckRunText = '';
+
+    const url = this.queueClient
+      .buildUrl(this.queueClient.getArtifact, taskId, runId, textArtifactName);
+
+    let res = await utils.throttleRequest({url, method: 'GET'});
+
+    if (res.status >= 400 && res.status !== 404) {
```

Comment:
(follow-up) I wonder if it would be useful if an artifact is explicitly specified, but not found, if mention should be made of it? On the other hand, I can imagine the default artifact being used most of the time.